org.thecrossingchurch.CustomJobs/Jobs/HubspotIntegrationPatching.cs

```diff
@@ -0,0 +1,442 @@
+﻿// <copyright>
+// Copyright by the Spark Development Network
+//
+// Licensed under the Rock Community License (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.rockrms.com/license
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+//
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using Quartz;
+using Rock;
+using Rock.Attribute;
+using Rock.Data;
+using Rock.Model;
+using Rock.Web.Cache;
+using Rock.Web.UI.Controls;
+using Newtonsoft.Json;
+using System.Net;
+using System.Reflection;
+using RestSharp;
+using Rock.Security;
+using System.ComponentModel;
+using System.Threading;
+using System.Diagnostics;
+using OfficeOpenXml.FormulaParsing.Excel.Functions.Text;
+using Newtonsoft.Json.Converters;
+using OfficeOpenXml.FormulaParsing.Excel.Functions.DateTime;
+using System.Data.Entity;
+using Rock.Search.Person;
+using Quartz.Impl.Matchers;
+using System.Text;
+using Rock.Lava.RockLiquid.Blocks;
+using RestSharp.Extensions;
+
+namespace org.crossingchurch.HubspotIntegration.Jobs
+{
+    /// <summary>
+    /// Job to supply hubspot contacts that already have rock_person_ids with other info.
+    /// </summary>
+    [DisplayName("Hubspot Integration: Update Records")]
+    [Description("This job only updates Hubspot contacts with a valid Rock ID with additional info from Rock.")]
+    [DisallowConcurrentExecution]
+
+    [TextField("AttributeKey", "The attribute key for the global attribute that contains the HubSpot API Key. The attribute must be encrypted.", true, "HubspotAPIKeyGlobal")]
+    [TextField("Business Unit", "Hubspot Business Unit value", true, "0")]
+    [DefinedValueField("Contribution Transaction Type",
+        AllowMultiple = false,
+        AllowAddingNewValues = false,
+        DefaultValue = Rock.SystemGuid.DefinedValue.TRANSACTION_TYPE_CONTRIBUTION,
+        DefinedTypeGuid = Rock.SystemGuid.DefinedType.FINANCIAL_TRANSACTION_TYPE
+    )]
+    [BooleanField("Include TMBT", defaultValue: false)]
+    [AccountField("Financial Account", "If syncing a total amount given which fund should we sync from")]
+    public class HubspotIntegrationPatching : IJob
+    {
+        private string key { get; set; }
+        private List<HSContactResult> contacts { get; set; }
+        private int request_count { get; set; }
+        private string businessUnit { get; set; }
```

Comment:
The names of these properties violate naming rules. Please update them.
- `key` -> `Key`
- `contacts` -> `Contacts`
- `request_count` -> `RequestCount`

And `businessUnit` has 0 references. So can we remove it?